### PR TITLE
Fix pageorder

### LINKED ISSUE #9: Integrate paging direction, when known
Some items have paging direction in the IA metadata, including when they should be R-L instead of L-R which is the default for many viewers.
An example: 
https://archive.org/details/Maslak-ul-akhyar-PersianManuscript/
https://archive.org/metadata/Maslak-ul-akhyar-PersianManuscript/

the metadata contains the following we can use:
`page-progression: "rl"`

### PR DESCRIPTION
**Pull request by hadro**
This fixes #9, but should be merged in after #22

### CODE REVIEW

**Comment by glenrobson** [comment]
It looks like the tests are failing. Is this due to a fix required in #22 before you can get the /iiif/3/rbmsbk_ap2-v4_2001_V55N4/manifest.json manifest?

**Comment by glenrobson** [comment]
Although its working in the public version:

https://iiif.archive.org//iiif/3/rbmsbk_ap2-v4_2001_V55N4/manifest.json

**Comment by hadro** [comment]
That's the wrong test, it's the next one that is throwing the error (`test_page_behavior_r_to_l`, not `test_page_behavior`), and that manifest currently throws an error on the live site: 
https://iiif.archive.org/iiif/nybc314063/manifest.json

And yes, that is one of the bugs that will be fixed by #22, it's the issue described in #14

So once #22 is ready to go, this can go ahead assuming you think it's all fine

File: iiify/resolver.py [hunk 1]
```diff
@@ -387,6 +387,25 @@ def create_manifest3(identifier, domain=None, page=None):
                     #                            id=f"https://iiif.archivelab.org/iiif/{identifier}${pageCount}/canvas",
                     #                            label=f"{page['leafNum']}")
                     pageCount += 1
+    
+
+            # Setting logic for paging behavior and starting canvases
+            # Start with paged (default) or individual behaviors
+            try:
```
Comment (glenrobson) [inline iiify/resolver.py:394]: Maybe add comment saying that it won't be set if the book reader brOptions has something other than mode/1up

File: iiify/resolver.py [hunk 2]
```diff
@@ -387,6 +387,25 @@ def create_manifest3(identifier, domain=None, page=None):
...
+                if bookreader['data']['brOptions']['defaults'] == "mode/1up":
+                    manifest.behavior = "individuals"
+            except:
+                manifest.behavior = "paged"
+            
+            # Then set left-to-right or right-to-left if present
```
Comment (glenrobson) [inline iiify/resolver.py:400]: Add test to see if pageProgression key exists in brOptions or try catch Key not found error.

**Pull request merged**